E3SM-1-0-LE/historical_r1i2p2f1.json

```diff
@@ -0,0 +1,103 @@
+
+  "source_type": "AOGCM AER",
+
+  "#note_experiment_id": "CMIP6 valid experiment_ids are found in CMIP6_CV.json",
+
+  "experiment_id": "historical",
+
+  "activity_id": "CMIP",
+
+  "sub_experiment_id": "none",
+
+  "realization_index": "1",
+
+  "initialization_index": "2",
+
+  "physics_index": "2",
+
+  "forcing_index": "1",
+
+  "#note_run_variant": "Text stored in attribute variant_info (recommended, not required description of run variant)",
+
+  "run_variant": "",
+
+  "parent_experiment_id": "piControl",
+
+  "parent_activity_id": "CMIP",
+
+  "parent_source_id": "E3SM-1-0",
+
+  "parent_variant_label": "r1i1p1f1",
+
+  "parent_time_units": "days since 0001-01-01",
+
+  "branch_method": "standard",
+
+  "branch_time_in_child": 0.0,
+
+  "#note_branch_time_in_parent":"i.e. first historical ensemble branch from 0101-01-01, gives branch time 36500 days. This value is a multiple of 365" ,
+  
+  "branch_time_in_parent": 36500.0,
+
+  "#note_institution_id": "institution_id must be registered at https://github.com/WCRP-CMIP/CMIP6_CVs/issues/new ",
+
+  "institution_id": "UCSB",
+
+  "#note_source_id": "source_id (model name) must be registered at https://github.com/WCRP-CMIP/CMIP6_CVs/issues/new ",
+
+  "source_id": "E3SM-1-0",
+
+  "calendar": "noleap",
+
+  "grid": "data regridded to a CMIP6 standard 1x1 degree lonxlat grid from the native grid using an area-average preserving method.",
+
+  "grid_label": "gr",
+
+  "nominal_resolution": "100 km",
+
+  "#output": "Root directory for output (can be either a relative or full path)",
+
+  "outpath": "CMIP6",
+
+  "#note_optional": " **** The following descriptors are optional and may be set to an empty string ",
+
+  "project PI": "Dave Bader (bader2@llnl.gov)",
+
+  "data contact": "E3SM-DATA-SUPPORT@LLNL.GOV",
+
+  "license": "CMIP6 model data produced by E3SM-Project is licensed under a Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file) and at https://pcmdi.llnl.gov/. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+
+  "#note_history": "include the original case name",
+
+  "history": "v2.LR.historical_0101",
+
+  "comment": "",
+
+  "references": "Golaz, J.-C., L. P. Van Roekel, X. Zheng and co-authors, 2022: The DOE E3SM Model Version 2: Overview of the physical model and initial model evaluation. JAMES, doi: 10.1029/2022MS003156; http://e3sm.org",
```

Comment:
@samanthastevenson please provide the reference paper to be included.

Reply:
The reference paper still needs to be resubmitted: it will be 
Stevenson, S., Huang, X., Zhao, Y., Di Lorenzo, E., Newman, M., van Roekel, L., Xu, T. and Capotondi, A., 2023

but title and possibly journal may still change

Reply:
Okay, we will use this as placeholder and can change after the reference is finalized. We will shoot for have the data out after paper is accepted.